### DISCUSSION TITLE
Export internal "Presence" for general use

### DISCUSSION THREAD

**Discussion by sondh0127** [unovue/reka-ui#963] [General]
Export internal "Presence" for general use

There are some use-cases for building open-hide animations that are best suited for Presence and data-state 
```
<Presence :present="open">
  <div v-bind="$attrs" ref="node" :data-state="open ? 'open' : 'closed'">
    Content
  </div>
</Presence>
```

**Comment by zernonia** [comment]
Yeah we can definitely export this component 😁

**Reply by sondh0127** [reply]
please consider update for upcoming release thank you

**Reply by zernonia** [reply]
As we are adjusting `Presence` component in v2 (to support SSR rendering, but visually hidden), this will only be expose in v2 ya 😁
